examples/extensions/r2/hello_r2.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+#
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+#
+
+import sys
+sys.path.append('..')
+
+from qiling import Qiling
+from qiling.const import QL_VERBOSE
+from qiling.extensions.r2 import R2
+
+
+def func(ql: Qiling, *args, **kwargs):
+    ql.os.stdout.write(b"=====hooked main=====!\n")
+    return
+
+def my_sandbox(path, rootfs):
+    ql = Qiling(path, rootfs, verbose=QL_VERBOSE.DEFAULT)
+    r2 = R2(ql)
+    assert(ql.loader.images[0].base == r2.baddr)
```

Comment:
I don't think `assert` is a good style for examples. They should be something in tests.

Reply:
Agree; I think a comment saying "expecting _x_ to be equal to _y_" is more appropriate here.